src/PrestaShopBundle/Security/Admin/EmployeeProvider.php

```diff
@@ -61,7 +61,7 @@ public function loadUserByUsername($username)
         if (isset($this->legacyContext->employee) && $this->legacyContext->employee->email == $username) {
             $employee = new Employee($this->legacyContext->employee);
             $employee->setRoles(
-                Access::getRoles($this->legacyContext->employee->id_profile)
+                array_merge(['ROLE_EMPLOYEE'], Access::getRoles($this->legacyContext->employee->id_profile))
```

Comment:
we need to add - at least - a minimal role or the User Provider will throw an exception!

This is not the case if you create a new profile and set no authorization at all: really uncommon, but still, this shouldn't throw a PHP exception :+1:

Reply:
Maybe use const for roles? (Don't know it is for others)

Reply:
others are values in bdd, I can add a constant in EmployeeProvider.

Reply:
fixed :+1: